actors/builtin/miner/miner_test.go

```diff
@@ -731,6 +777,14 @@ func (h *actorHarness) submitWindowPost(rt *mock.Runtime, deadline *miner.Deadli
 	rt.SetCaller(h.worker, builtin.AccountActorCodeID)
 	rt.ExpectValidateCallerAddr(h.worker)
 
+	reward := big.NewIntUnsigned(1e18)
```

Comment:
Aside: we might move some of these "network" properties into the harness so they can easily be consistent between calls.